a3p-integration/proposals/a:upgrade-next/ante-fees.test.js

```diff
@@ -0,0 +1,72 @@
+import test from 'ava';
+
+import { CHAINID, GOV1ADDR, GOV2ADDR, agd } from '@agoric/synthetic-chain';
+
+test(`ante handler sends fee only to vbank/reserve`, async t => {
+  const [feeCollector, vbankReserve] = await Promise.all(
+    ['fee_collector', 'vbank/reserve'].map(async name => {
+      const {
+        account: {
+          '@type': moduleAcct,
+          base_account: { address },
+        },
+      } = await agd.query('auth', 'module-account', name);
+
+      t.is(
+        moduleAcct,
+        '/cosmos.auth.v1beta1.ModuleAccount',
+        `${name} is a module account`,
+      );
+      return address;
+    }),
+  );
+
+  const getBalances = () =>
+    Promise.all(
+      [feeCollector, vbankReserve].map(async address => {
+        const { balances } = await agd.query('bank', 'balances', address);
+        return balances;
+      }),
+    );
+
+  const [feeCollectorStartBalances, vbankReserveStartBalances] =
+    await getBalances();
```

Comment:
Suggestion for readability later in the function:
```suggestion
  const getBalances = addresses =>
    Promise.all(
      addresses.map(async address => {
        const { balances } = await agd.query('bank', 'balances', address);
        return balances;
      }),
    );

  const [feeCollectorStartBalances, vbankReserveStartBalances] =
    await getBalances([feeCollector, vbankReserve]);
```